src/utils/utils.js

```diff
@@ -869,6 +869,12 @@ export function fieldData(data, component) {
     if (component.multiple && !Array.isArray(data[component.key])) {
       data[component.key] = [data[component.key]];
     }
+
+    // Fix for checkbox type radio submission values in tableView
+    if (component.type === 'checkbox' && component.inputType === 'radio') {
+      return !!data[component.name];
```

Comment:
```data[component.name]``` will be a string that will be set to the ```component.value```.  You need to also check for component.value I believe like this.

```
return data[component.name] === component.value;
```

Reply:
Makes sense! Missed that thing, thanks for catching.